bootstrapper/cards/cardsNg2Bootstrapper.ts

```diff
@@ -59,7 +60,7 @@ export class CardsBootstrapper {
 		this.options = [1, 2, 3, 4, 5];
 
 		this.builder = cardContainerBuilder;
-		this.builder.dataSource.buildSimpleDataSource(items);
+		this.builder.dataSource.buildDataServiceDataSource<ICardItem>(() => this.wait(items));
```

Comment:
Could this be better accomplished by using a operator such as timeout instead of a custom wait function?